frontend/templates/express/src/app.js.hbs

```diff
@@ -0,0 +1,120 @@
+require("express");
+require("express-async-errors");
+
+const path = require("path");
+const session = require("express-session");
+const AWS = require("aws-sdk");
+const DynamoDBStore = require("connect-dynamodb")(session);
+
+const commonExpress = require("di-ipv-cri-common-express");
+
+const setHeaders = commonExpress.lib.headers;
+const setScenarioHeaders = commonExpress.lib.scenarioHeaders;
+const setAxiosDefaults = commonExpress.lib.axios;
+
+const { setAPIConfig, setOAuthPaths } = require("./lib/settings.js");
+const { setGTM } = require("di-ipv-cri-common-express/src/lib/settings");
+const { getGTM } = require("di-ipv-cri-common-express/src/lib/locals");
+const { setI18n } = require("di-ipv-cri-common-express/src/lib/i18next");
+
+const {
+  API,
+  APP,
+  PORT,
+  SESSION_SECRET,
+  SESSION_TABLE_NAME,
+  SESSION_TTL,
+} = require("./lib/config.js");
+
+const { setup } = require("hmpo-app");
+
+const loggerConfig = {
+  console: true,
+  consoleJSON: true,
+  app: false,
+};
+
+AWS.config.update({
+  region: "eu-west-2",
```

Comment:
Do we want this region to be hard-coded? Most places in `-api` repos we try and make it a reference

Reply:
Updated.